meshroom/core/node.py

```diff
@@ -1334,7 +1352,7 @@ def attributeDescFromName(refAttributes, name, value, strict=True):
 
         return None
 
-    def _addAttribute(self, name, val, isOutput):
+    def _addAttribute(self, name, val, isOutput, internalAttr=False):
```

Comment:
Need to declare the param in the documentation.

Reply:
Done